### PR TITLE
Add variable to configure maxHttpHeaderSize attribute

### PR DESCRIPTION
**Pull request by danifr**
#### Pull Request (PR) description
Add variable to be able to configure tomcat's maxHttpHeaderSize attribute.

Default is set to 8192 (from https://tomcat.apache.org/tomcat-9.0-doc/config/http.html `maxHttpHeaderSize: If not specified, this attribute is set to 8192 (8 KB).`)

### CODE REVIEW

**Comment by danifr** [comment]
Hello @bastelfreak, can you please check this PR when you have some time? Thanks.

**Review by kenyon — APPROVED** [review]

**Comment by danifr** [comment]
I will merge this change if no one objects to

File: manifests/config.pp [hunk 1]
```diff
@@ -33,12 +34,13 @@
   }
 
   if $manage_server_xml == 'augeas' {
-    $_tomcat_max_threads   = { maxThreads   => $tomcat_max_threads }
-    $_tomcat_accept_count  = { acceptCount  => $tomcat_accept_count }
-    $_tomcat_port          = { port         => $tomcat_port }
-    $_tomcat_redirect_port = { redirectPort => $tomcat_redirect_port }
+    $_tomcat_max_threads          = { maxThreads   => $tomcat_max_threads }
+    $_tomcat_max_http_header_size = { redirectPort => $tomcat_max_http_header_size }
```
Comment (smortex) [inline manifests/config.pp:38]: Isn't it supposed to be:

```suggestion
    $_tomcat_max_http_header_size = { maxHttpHeaderSize => $tomcat_max_http_header_size }
```

It looks like this augeas thing is not tested

**Review by smortex — CHANGES_REQUESTED** [review]
Hey! Sorry for missing the mention, I guess I had to sort out too many notifications from GitHub when I came back from holidays :unamused:…

I think I found a typo, can you double-check it?

File: manifests/config.pp (hunk 1 above)
Comment (danifr) [reply manifests/config.pp:38]: OMG, yes. Too much copy pasting... sorry I will fix it now

**Review by smortex — APPROVED** [review]
LGTM!

**Pull request merged**